### PR TITLE
Brian Post #5

### PR DESCRIPTION
**Pull request by mastastealth**
Tada.

### CODE REVIEW

File: _posts/2016-11-04-a-yearish-in-ember.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "A Year-ish in Ember"
+date: 2016-11-04 06:00:00
```
Comment (laurawhalin) [inline _posts/2016-11-04-a-yearish-in-ember.md]: Can you update this date and the filename to today's date? I'll get this put out today.

File: _posts/2016-11-04-a-yearish-in-ember.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+I repeatedly compare Ember to [Meteor](https://www.meteor.com/), seeing as how the latter was the first framework I really got the hang of (I had tried Angular before that, and also had trouble with it). Pumping out Meteor apps felt so much easier to me, and learning it definitely did not feel as burdensome. There are many differences between the 2 frameworks of course, Ember's opinionated structure for example is a double-edged sword that helps keep code consistent yet adds another layer of things to learn before stuff actually works.
```
Comment (laurawhalin) [inline _posts/2016-11-04-a-yearish-in-ember.md]: Can you put commas around `, for example,` for easier readability?

File: _posts/2016-11-04-a-yearish-in-ember.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## The Ugly
+
+Debugging Ember issues isn't particularly fun. To be fair, it seems Javascript isn't as debug friendly as other languages out there, however when working with just vanilla JS I find it fairly simple to see where something went wrong, and why. Although Ember does link to some handy guides/posts with some of their error messages (like deprecation warnings), sometimes even using the link gives me no real understanding of how to fix my problem, or where it even arose in my code. [Ember Inspector](https://github.com/emberjs/ember-inspector) has come in handy, but also feels a little rough on the edges at time, especially when trying to dig deeper into components and what's being passed (or not) through them.
```
Comment (laurawhalin) [inline _posts/2016-11-04-a-yearish-in-ember.md]: `rough on the edges at times` (add s to time)

**Comment by mastastealth** [comment]
Done @laurawhalin 👍

**Pull request merged**